noxfile.py

```diff
@@ -129,14 +154,28 @@ def system(session):
         "google-cloud-iam",
         "google-cloud-pubsub < 2.0.0",
         "google-cloud-kms < 2.0dev",
+        "-c",
+        constraints_path,
```

Comment:
Ugh, this change is now in conflict with #433, which works around the breakage incurred by `googleapis-common-protos` dropping Python 2.7 support.  Likely we need to tweak `synth.py` to preserve that change.

Reply:
Done.